equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/util/DateUtil.java

```diff
@@ -43,4 +47,18 @@ public static Date convertStringToSimpleDate(String dateToConvert) throws ParseE
 
         return new SimpleDateFormat(DATE_FORMAT_PATTERN, Locale.UK).parse(dateToConvert);
     }
-}
+
+    public static String calculateTimeDifference(String startDateTime, String endDateTime, String format) {
+        DateFormat dateFormat = new SimpleDateFormat(format);
+        try {
+            Date startDate = dateFormat.parse (startDateTime);
+            Date endDate = dateFormat.parse (endDateTime);
+            long timeDiff = endDate.getTime() - startDate.getTime();
+            return DurationFormatUtils
+                    .formatDuration(timeDiff, "HH:mm:ss", true);
```

Comment:
`HH:mm:ss` - Can you put this at the top in a String final variable?

Reply:
done